# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/hyprland/default-term.patch
# Copyright (C) 2024 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- Hyprland-0.50.0/example/hyprland.conf.vanilla	2025-07-16 15:38:55.390002492 +0200
+++ Hyprland-0.50.0/example/hyprland.conf	2025-07-16 15:39:10.560222538 +0200
@@ -25,7 +25,7 @@
 # See https://wiki.hypr.land/Configuring/Keywords/
 
 # Set programs that you use
-$terminal = kitty
+$terminal = alacritty
 $fileManager = dolphin
 $menu = wofi --show drun
 
